static/js/alert_words.js

```diff
@@ -32,8 +32,8 @@ export function process_message(message) {
 
     for (const word of my_alert_words) {
         const clean = _.escapeRegExp(word);
-        const before_punctuation = "\\s|^|>|[\\(\\\".,';\\[]";
-        const after_punctuation = "\\s|$|<|[\\)\\\"\\?!:.,';\\]!]";
+        const before_punctuation = "\[^\\/\\w]|^|>|[\\(\\\".,';\\[]";
+        const after_punctuation = "\[^\\/\\w]|$|<|[\\)\\\"\\?!:.,';\\]!]";
```

Comment:
Maybe write these using raw strings rather than the complicated escape logic?

Reply:
Yes, @timabbott, I tried it using raw strings but it requires look-ahead and look-behind which works fine but gave errors in tests. So I used the escape-logic instead.